src/components/compose.jsx

```diff
@@ -226,11 +226,11 @@ function Compose({
   const {
     statuses: {
       maxCharacters,
-      maxMediaAttachments,
+      maxMediaAttachments,  // Beware: it can be undefined on *oma instances!
```

Comment:
Don't think the comment need to specify "on *oma instances" here as it can be any instances.

Reply:
```suggestion
      maxMediaAttachments,  // Beware: it can be undefined!
```

Here we go!